Neos.Flow/Classes/Error/Debugger.php

```diff
@@ -197,9 +197,8 @@ protected static function renderObjectDump($object, int $level, bool $renderProp
         $scope = '';
         $additionalAttributes = '';
 
-        if ($object instanceof \Doctrine\Common\Collections\Collection || $object instanceof \ArrayObject) {
-            // The doctrine Debug utility usually returns a \stdClass object that we need to cast to array.
-            return self::renderArrayDump((array)\Doctrine\Common\Util\Debug::export($object, 3), $level, $plaintext, $ansiColors);
+        if (is_iterable($object)) {
+            return self::renderArrayDump($object, $level, $plaintext, $ansiColors);
```

Comment:
Had to check myself too, but apparently `Doctrine\Common\Util\Debug` is deprecated anyway in favor of symfony/var-dumper. Also, the only special thing that Doctrine debug util did in case of Collections, was convert them to an array via `->toArray()`, which should not be necessary. It had a couple of special additions for DateTime and Proxy classes though, but those also didn't effect this code path AFAIS.